package.json

```
@@ -70,7 +70,7 @@
     "@formatjs/intl-relativetimeformat": "^5.2.6",
     "@fortawesome/fontawesome-free": "^5.13.0",
     "@material-ui/core": "^4.11.0",
-    "@metamask/controllers": "^4.0.0",
+    "@metamask/controllers": "4.0.1",
```

I'm guessing that dropping the `^` was a mistake?
```suggestion
    "@metamask/controllers": "^4.0.1",
```

---

This'll require a lockfile update as well